### PR TITLE
[WIP] experimental new structure

### PR DESCRIPTION
**Pull request by Rich-Harris**
See sveltejs/sapper#157

### CODE REVIEW

**Comment by Rich-Harris** [comment]
It's not working because the new version of Sapper that it depends on hasn't been released yet.

File: package.json [hunk 1]
```diff
@@ -6,19 +6,21 @@
     "dev": "sapper dev",
     "build": "sapper build",
     "export": "sapper export",
-    "start": "sapper start",
+    "start": "node build",
```
Comment (silentworks) [inline package.json]: Is this correct? shouldn't this be `npm build`?

Comment (Rich-Harris) [reply package.json]: it's correct, yeah — as of 0.11, Sapper creates a launcher file which defaults to `build/index.js`. So to run the app you just to `node build`

Comment (arxpoetica) [reply package.json]: This probably explains my problems w/ trying to build when upgrading Sapper. ;) Need to upgrade `sapper-template` too...

File: app/App.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<Nav page={
```
Comment (arxpoetica) [inline app/App.html]: @Rich-Harris why the intensive `props.path` logic here? This seems very **manifest**-y, which goes against the grain of the automated path-finding you seem to support? What exactly is the function of these page props?

Comment (arxpoetica) [reply app/App.html]: In other words, do I have to add to this for every page I add?

Comment (Rich-Harris) [reply app/App.html]: you can pass `props.path` down to `<Nav>` instead (in fact, I prefer that — will change it)...

```html
<Nav path={props.path}/>
```

...but yes, you have to explicitly pass stuff down, just like you used to have to explicitly pass down a `page` property via the `<Layout>` component in the previous template. How else will the nav bar know which option to mark as active?

File: package.json (hunk 1 above)
Comment (arxpoetica) [reply package.json]: I got it all working.

**Comment by arxpoetica** [comment]
That's much better, thx.

**Pull request merged**